### PR TITLE
build(ci): multi-platform build distribution with bake

### PR DESCRIPTION
**Pull request by crazy-max**
### Proposed changes

similar to https://github.com/docker/docs/pull/17180 but with bake.

### CODE REVIEW

File: content/build/ci/github-actions/multi-platform.md [hunk 1]
```diff
@@ -173,4 +172,183 @@ jobs:
...
           docker buildx imagetools inspect ${{ env.REGISTRY_IMAGE }}:${{ steps.meta.outputs.version }}
-```
\ No newline at end of file
+```
+
+### With bake
+
+You can also do the same as the previous example using the [bake action](https://github.com/docker/bake-action).
+A live example [on this repo](https://github.com/crazy-max/docker-linguist) is
+available as well.
```
Comment (dvdksn) [inline content/build/ci/github-actions/multi-platform.md:181]:
```suggestion
### With Bake

It's also possible to build on multiple runners using Bake, with the
[bake action](https://github.com/docker/bake-action).
You can find a live example [in this GitHub repository](https://github.com/crazy-max/docker-linguist).

The following example achieves the same results as described in
[the previous section](#distribute-build-across-multiple-runners).
```

**Review by dvdksn — COMMENTED** [review]
I haven't tried the workflow but I'm sure it works :-D Just have some suggestions for editorial updates.

**Review by dvdksn — DISMISSED** [review]

**Comment by usha-mandya** [comment]
@crazy-max Looks like there's conflicting file. Could you PTAL?

**Comment by crazy-max** [comment]
Done

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**